literature/Engstfeld_2018_polycristalline_17743/Engstfeld_2018_polycrystalline_17743_4b_1.json

```diff
@@ -0,0 +1 @@
+{"version": 1, "source": {"doi": "https://doi.org/10.1002/chem.201803418", "bib": "Engstfeld_2018_Polycrystalline_17747", "figure": "4b", "curve": 1}, "curator": {"name": "Albert Engstfeld", "orcid": "https://orcid.org/0000-0002-9686-3948", "digitized": null}, "figure description": {"linked measurements": null, "comment": "", "type": "digitized", "measurement type": "CV", "scan rate": {"value": 50.0, "unit": "mV / s"}, "potential scale": {"unit": "V", "reference": "RHE"}, "current": {"unit": "uA / cm2"}}, "electrochemical system": {"electrolyte": {"type": "aq", "components": [{"name": "water", "sum formula": "H2O", "type": "solvent", "proportion": {"value": 100, "unit": "volume percent"}, "source": {"refinement": "Millipore MilliQ", "quality": "ultrapure water", "total organic carbon": {"value": "none", "unit": "none"}}}, {"name": "potassium hydroxide", "sum formula": "KOH", "type": "alkaline", "source": {"supplier": "Merck Suprapur", "LOT": null}, "concentration": {"value": 0.1, "unit": "M"}}], "ph": {"value": 13, "uncertainty": "none"}, "temperature": {"value": 298.15, "unit": "K"}}, "electrodes": {"configuration": "three", "reference electrode": {"type": "RHE", "source": {"supplier": "homemade"}}, "counter electrode": {"material": "Au", "crystallographic orientation": "poly", "shape": "mesh"}, "working electrode": {"material": "Cu", "crystallographic orientation": 100, "source": {"supplier": "Mateck", "LOT": null}, "shape": {"type": "head shaped", "height": {"value": 2, "unit": "mm"}, "diameter": {"value": 4.4, "unit": "mm"}}, "geometric electrolyte contact area": {"value": null, "unit": "cm-2"}, "preparation procedure": "sputter and heating under UHV conditions"}}}, "profile": "tabular-data-package", "resources": [{"path": "Engstfeld_2018_polycrystalline_17743_4b_1.csv", "profile": "tabular-data-resource", "name": "Engstfeld_2018_polycrystalline_17743_4b_1", "format": "csv", "mediatype": "text/csv", "encoding": "utf-8", "schema": {"fields": [{"name": "t", "type": "number", "format": "default"}, {"name": "U", "type": "number", "format": "default"}, {"name": "j", "type": "number", "format": "default"}], "missingValues": [""]}}]}
```

Comment:
I thought we did not want to put all these files into the repository but instead generate them in the workflow.